src/Sylius/Bundle/ChannelBundle/composer.json

```diff
@@ -27,7 +27,9 @@
         "symfony/framework-bundle": "^2.7.7"
     },
     "require-dev": {
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6",
```

Comment:
Wouldn't be more appropriate to have this next to PHPUnit? We do not sort deps alphabetically.